modules/reference/pages/feature/Jakarta-EE9-feature-updates.adoc

```diff
@@ -0,0 +1,220 @@
+// Copyright (c) 2021 IBM Corporation and others.
+// Licensed under Creative Commons Attribution-NoDerivatives
+// 4.0 International (CC BY-ND 4.0)
+// https://creativecommons.org/licenses/by-nd/4.0/
+//
+// Contributors:
+// IBM Corporation
+//
+// This doc is hosted in the Red Hat Runtimes documentation. Any changes made to this doc also need to be made to the version that's located in the PurpleLiberty GitHub repo (https://github.com/PurpleLiberty/docs).
+//
+:page-description: For Jakarta EE 9 applications to run in Open Liberty, Jakarta EE 9 features must be specified in the server configuration. If Jakarta EE 8 feature name is incorrectly used with the Jakarta EE 9 version, the server does not start, and the server logs provide information to say which new feature name must be used.
+:seo-title: Jakarta EE9 feature updates - OpenLiberty.io
+:projectName: Open Liberty
+:page-layout: Jakarta EE 9
+:page-type: feature updates
+= Jakarta EE 9 feature updates
+
+For Jakarta EE 9 applications to run in Open Liberty, Jakarta EE 9 features must be specified in the server configuration. If Jakarta EE 8 feature name is incorrectly used with Jakarta EE 9 version, the server does not start, and the server logs provide information to say which new feature name must be used.
+
+In many cases, Jakarta EE 9 features use the same short names as the corresponding Jakarta EE 8 features. In those cases, the Jakarta EE 9 features simply have an updated feature version.
+For example, the Contexts and Dependency Injection feature is updated from `cdi-2.0` to `cdi-3.0`.
+
+However, in some cases, the feature short name is also different, as with the Enterprise JavaBeans 3.2 feature `ejb-3.2`, which was updated to Enterprise Beans 4.0 `enterpriseBeans-4.0`.
```

Comment:
use parentheses for the short names here eg

the Enterprise JavaBeans 3.2 feature (`ejb-3.2`)